loco-gen/src/tera_ext.rs

```diff
@@ -181,59 +182,78 @@ impl tera::Function for FormField {
     </div>"#
                 )
             }
-            "Vec<f32>" | "Option<Vec<f32>>" => {
+            "Vec<i32>" | "Option<Vec<i32>>" => {
                 let edit_input = input_number(
                     fname,
                     "{{val}}",
                     is_required,
                     input_class,
-                    Some((f32::MIN, f32::MAX)),
-                    Some(r#"custom_type="array" step="0.1""#),
+                    Some((i32::MIN, i32::MAX)),
+                    Some(r#"custom_type="array""#),
```

Comment:
Why was `step="0.1"` removed?
Did you try generating the scaffold with the UI and testing it?